src/data/Product.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import "./Products"
+import formatPrice from '../helpers/formatPrice'
+
+function Product ({product,handleClick}) {
+    const {name,price, img, description} = product
+    return (
+        <div>
+            <h3>{name}</h3>
+            <p>Price: {formatPrice(price)}</p>
+            {"\n"}
```

Comment:
What's going on here? I can guess maybe why you put this here, but the better way is to use CSS to put a margin or padding between these elements.